bin/picca_Pk1D.py

```diff
@@ -200,9 +200,9 @@ def process_all_files(index_file_args):
                                            linear_binning=False)
 
             # Compute pk_noise
-            run_noise = False
-            if args.noise_estimate == 'pipeline':
-                run_noise = True
+            run_noise = True
```

Comment:
I will remove that, because it is only provisory for me to compare diff and pipeline noise

Reply:
Ideally you'd have that as a mode to run the pipeline instead of removing this completely. That way future diff vs pipeline tests won't require more changes and you can say which version exactly you used in the paper...